packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2664,6 +2664,67 @@ class SizedOverflowBox extends SingleChildRenderObjectWidget {
 /// needed, prefer removing the widget from the tree entirely rather than
 /// keeping it alive in an [Offstage] subtree.
 ///
+/// {@tool snippet}
```

Comment:
This should be using `dartpad` instead of `snippet` to make the example interactive in our API docs.